election/settings.py

```diff
@@ -13,7 +13,13 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost', '127.0.0.1']
```

Comment:
We don't need the blockchain URL to be added to the allow host.

Reply:
its defunct , will be cleared in next pr since ajax communication is no longer needed with the current implementation